README.md

```diff
@@ -31,63 +42,153 @@ envs:
+| `subdomain`                        | _optional_  | The name of the custom subdomain set to access your Datadog application. If the URL used to access Datadog is `myorg.datadoghq.com`, the `subdomain` value needs to be set to `myorg`.                                |
+| `testSearchQuery`                  | _optional_  | Trigger tests corresponding to a [search][C-5]  query. This can be useful if you are tagging your test configurations. See [best practices][C-7]  for more information on tagging.                                    |
+| `tunnel`                           | _optional_  | Enable [tunnel][C-4] to interact with Datadog API.                                                                                                                                                                    |
+| `variables`                        | _optional_  | Key-value pairs for injecting variables into tests. Must be formatted using `KEY=VALUE`.                                                                                                                              |
+
+## Further reading
+
+Additional helpful documentation, links, and articles:
+
+- [Continuous Testing and CI/CD Configuration][C-1]
+- [Best practices for continuous testing with Datadog][E-1]
+
+<!-- Links to Marketplace -->
+[A-1]: https://marketplace.visualstudio.com/items?itemName=Datadog.datadog-ci
+
+<!-- Github links -->
+[B-1]: https://github.com/DataDog/datadog-ci/blob/master/.github/workflows/e2e/global.config.json
+[B-2]: https://github.com/DataDog/datadog-ci/tree/master/src/commands/synthetics#test-files
+[B-3]: https://github.com/bitrise-io/bitrise
+
+<!-- Links to datadog documentation -->
+[C-1]: https://docs.datadoghq.com/continuous_testing/cicd_integrations/configuration
+[C-2]: https://docs.datadoghq.com/continuous_testing/cicd_integrations/configuration/?tab=npm#test-files
+[C-3]: https://docs.datadoghq.com/continuous_testing/cicd_integrations/configuration/?tab=npm#global-configuration-file-options
+[C-4]: https://docs.datadoghq.com/continuous_testing/testing_tunnel
+[C-5]: https://docs.datadoghq.com/synthetics/search/#search
+[C-6]: https://docs.datadoghq.com/account_management/api-app-keys/
+[C-7]: https://docs.datadoghq.com/developers/guide/what-best-practices-are-recommended-for-naming-metrics-and-tags/#rules-and-best-practices-for-naming-tags
+
+<!-- Integration specific links -->
+[D-1]: https://devcenter.bitrise.io/en/steps-and-workflows/introduction-to-steps/adding-steps-to-a-workflow.html
+[D-2]: https://devcenter.bitrise.io/en/builds/secrets.html#setting-a-secret
+[D-3]: https://devcenter.bitrise.io/en/steps-and-workflows/introduction-to-steps/step-inputs.html
+
+<!-- Other -->
+[E-1]: https://www.datadoghq.com/blog/best-practices-datadog-continuous-testing/
```

Comment:
I didn't know that was possible! We have to ensure it's supported by our Datadog docs, since this README page will at some point be single-sourced (i.e. used as the source code for the Datadog docs as well).

[**Example:**](https://docs.datadoghq.com/continuous_testing/cicd_integrations/github_actions/)

![image](https://github.com/DataDog/synthetics-test-automation-bitrise-step-run-tests/assets/9317502/f97cc0a4-f1c4-4a83-8581-cfb06ef7c9ac)